### ISSUE TITLE
Windows 10, Edge, Fails to load stage

### ISSUE THREAD

**Issue by martinohanlon** [LLK/scratch-gui#1269]
## Expected Behavior

The default cat sprite should appear in the stage.

### Actual Behavior

Nothing appears in the stage and when the green flag is ticked nothing happens.

### Steps to Reproduce

Open [https://llk.github.io/scratch-gui/](https://llk.github.io/scratch-gui/) in Edge.

### Operating System and Browser

Windows 10
Microsoft Edge 41.16299.15.0
Microsoft EdgeHTML 16.16299

Errors reported by the developers tools:

```
 SCRIPT5007: Unable to get property 'rotationCenter' of undefined or null reference
raven.js (13,1)

 HTTP429: TOO MANY REQUESTS - The server is refusing to service the request because too many requests have been submitted by the client.
(XHR)POST - https://sentry.io/api/253297/store/?sentry_version=7&sentry_client=raven-js%2F3.20.1&sentry_key=dcc4ed76876b411c99997f4ad5ab78a5
```

Screenshots of scratch gui, get.webgl.org and webglreport.com:

![get_webgl_org_screen](https://user-images.githubusercontent.com/2329501/35031313-465f76cc-fb5a-11e7-974f-648331c9b91a.PNG)
![scratch_3_screenshot](https://user-images.githubusercontent.com/2329501/35031315-46d4b0c2-fb5a-11e7-87bc-be1b4959b6eb.PNG)
![webglreport_com_screen](https://user-images.githubusercontent.com/2329501/35031316-46f10f88-fb5a-11e7-972d-a03f52360bcc.PNG)

**Comment by thisandagain** [comment]
Hmm! That's interesting. We'll take a look today. It could potentially be related to #1250, but seems more likely related to `SCRIPT5007: Unable to get property 'rotationCenter' of undefined or null reference`. Can you test in the `develop` branch? I believe we may have resolved this in #1222 recently: https://llk.github.io/scratch-gui/develop

**Comment by martinohanlon** [comment]
This issues looks like its resolved in the develop branch.
![image](https://user-images.githubusercontent.com/2329501/35046191-3523ba88-fb8e-11e7-954e-a09c7fc6138b.png)
